amplify/backend/backend-config.json

```diff
@@ -274,295 +274,5 @@
       "providerPlugin": "awscloudformation",
       "service": "Lambda"
     }
-  },
```

Comment:
Not sure why the backend-config.json file has been edited in this PR.

Reply:
Every `npm ci` does this, we need to update this on master :/
Otherwise for now @CzarekDryl  run `git checkout master -- amplify/backend/backend-config.json` and update this PR :D